examples/simple-typescript/index.ts

```diff
@@ -0,0 +1,18 @@
+// You can use "import Josh from 'josh'" instead of this, this is just to access the library instead of installing it.
+import Josh from '../../index.js';
```

Comment:
I really think examples should be `from 'josh';` , since people have a tendency to copy/paste those and their initial trial will fail if it's a relative path ^_^